{
  "debianbug": 742898,
  "description": "The auth_token middleware in the OpenStack Python client library for Keystone (aka python-keystoneclient) before 0.7.0 does not properly retrieve user tokens from memcache, which allows remote authenticated users to gain privileges in opportunistic circumstances via a large number of requests, related to an \"interaction between eventlet and python-memcached.\"",
  "releases": {
    "bullseye": {
      "fixed_version": "1:0.6.0-4",
      "repositories": {
        "bullseye": "1:3.19.0-2"
      },
      "status": "resolved",
      "urgency": "low"
    },
    "buster": {
      "fixed_version": "1:0.6.0-4",
      "repositories": {
        "buster": "1:3.17.0-2"
      },
      "status": "resolved",
      "urgency": "low"
    },
    "jessie": {
      "fixed_version": "1:0.6.0-4",
      "repositories": {
        "jessie": "1:0.10.1-2+deb8u1"
      },
      "status": "resolved",
      "urgency": "low"
    },
    "sid": {
      "fixed_version": "1:0.6.0-4",
      "repositories": {
        "sid": "1:3.19.0-2"
      },
      "status": "resolved",
      "urgency": "low"
    },
    "stretch": {
      "fixed_version": "1:0.6.0-4",
      "repositories": {
        "stretch": "1:3.2.0-4"
      },
      "status": "resolved",
      "urgency": "low"
    }
  },
  "scope": "remote"
}